docs/releases/2.1.txt

```diff
@@ -196,6 +196,10 @@ Models
   plan of a queryset's query.
 
 *  :meth:`.QuerySet.raw` now supports :meth:`~.QuerySet.prefetch_related`.
+* Slicing notation added to F expressions over the following field-types:
```

Comment:
This is now incorrect since only `ArrayField` along with `CharField` and its derivatives ('EmailField',  'SlugField', 'URLField') has this functionality implemented. We should also consider `HStoreField` as mentioned in the ticket. I feel we need to review (and write tests for) which field types that can have this and not.

Reply:
It also needs to be moved to another changelog file.

Reply:
* HStoreField - Not sure if it would make sense since that is a dict-like structure which doesn't really have an order/indices.
* BinaryField / GenericIPAddressField - was incorrectly implemented using SUBSTRING in the original PR, which was untested and doesn't work in postgres.
* FilePathField / TextField - would be possible to implement using SUBSTRING.

Reply:
"field types" doesn't need an apostrophe

Reply:
Fixed and moved to 3.0 release notes.